lib/react_client/js_interop_helpers.dart

```diff
@@ -30,7 +30,7 @@ class _MissingJsMemberError extends Error {
 /// (see: https://github.com/dart-lang/sdk/issues/25053).
 ///
 /// __Defined in this package's React JS files.__
-final dynamic Function(dynamic jsObj, String key) getProperty = (() {
+final dynamic Function(dynamic jsObj, dynamic key) getProperty = (() {
```

Comment:
These could be moved back to string, they are left over from when i was attempting the jsbackedmap indicated by a symbol path...